.travis.yml

```diff
@@ -18,5 +18,14 @@ before_install:
   - sudo apt-get update
   - sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-engine
 
+install:
+  - docker-compose run web pip install -r requirements.txt
```

Comment:
This is [done during image build](https://github.com/AdventureLookup/adventurelookup-backend/blob/master/web/Dockerfile#L16).
I would suggest removing it from here and maybe putting the build step into install as it is a part of the setup process and not running tests themselves.